sections/schema.js

```diff
@@ -549,8 +549,8 @@ export default [
   {
     type: "method",
     method: "unique",
-    example: "table.unique(columns, [indexName])",
-    description: "Adds an unique index to a table over the given `columns`. A default index name using the columns is used unless indexName is specified.",
+    example: "table.unique(columns, [indexName]|options={[indexName:string],[deferrable:'not deferrable'|'immediate'|'deferred']})",
```

Comment:
object syntax is preferred, let's drop the old one so that it is easier to deprecate later

Reply:
Got it. i also added some example.